dev/breeze/doc/09_release_management_tasks.rst

```diff
@@ -788,6 +788,29 @@ default is to build ``both`` type of distributions ``sdist`` and ``wheel``.
   :width: 100%
   :alt: Breeze release-management prepare-airflow-ctl-distributions
 
+Preparing Airflow CTL Tarball
+"""""""""""""""""""""""""""""
+
+You can prepare Airflow CTL source tarball using Breeze:
+
+.. code-block:: bash
+
+     breeze release-management prepare-airflow-ctl-tarball
```

Comment:
We also don't need airflowctl source tarball, the wheel and sdist are enough like Task SDK.

breeze release-management prepare-task-sdk-distributions

Reply:
Thanks for the quick review Kaxil!